manifests/integreatly-amq-online/amq-online-1.3.1/amq-online-addressplans.crd.yaml

```diff
@@ -48,9 +48,9 @@ spec:
               type: object
               properties:
                 router:
-                  type: number
+                  type: string
```

Comment:
Do these changes need to go upstream too, for future releases? If so, can you make an issue in the technical debt epic to make sure we don't forget.

Reply:
The recommendation is to use `resource.Quantity` when type is number, although I tried that and it didn't work so I went for type string instead. I'd say we better create a ticket to look into it further.